packages/nuejs.org/docs/tags.md

```diff
@@ -87,25 +87,25 @@ Images can link to URL's with `href` attribute:
 [.options]
   #### [image] options
 
-  `alt` is an alternate text for the image
+  `alt` is an alternate text for the image.
 
-  `src` image source
+  `src` image source.
 
-  `caption` image caption
+  `caption` image caption.
 
-  `href` a link URL for the image
+  `href` a URL link to the image.
```

Comment:
```suggestion
  `href` adds a link to the image.
```

Feels like it's duplicated, because at least in my surroundings, URL and link are used interchangeably

Also, it's not a link TO the image itself (would be `src`), but to another location

I find it quite challenging to provide a short description that properly explains that it points to another URL instead of to the image itself.

Please let me know of better descriptions.

Reply:
`href` adds a link on the image. Your change reads much better. great.